knotx-example/knotx-example-monolith/src/test/java/com/cognifide/knotx/monolith/SampleApplicationTest.java

```diff
@@ -73,37 +77,45 @@ public void localMultipleFormWithGetTest(TestContext context) {
 
   @Test
   public void localMultipleFormWithPostTest(TestContext context) {
```

Comment:
Could you rename test methods in order to meet our [naming convention](https://github.com/Cognifide/knotx/blob/master/CONTRIBUTING.md#tests-naming-convention)?
E.g. `whenRequestingMultipleFormWithPost_expectResultWithPostRequestProcessed` or smth similar.